Clarify automatic dataset downloads in training docs (#24155)
ultralytics/ultralytics: docs/en/modes/train.md

    * YOLO26 datasets like COCO, VOC, ImageNet, and many others automatically download on first use, i.e., `yolo train data=coco.yaml`

!!! tip "Automotive and dashcam-style footage"

>     For vehicle-centric detection, you can start from configs such as [KITTI](https://docs.ultralytics.com/datasets/detect/kitti/) (`kitti.yaml`) or [VisDrone](https://docs.ultralytics.com/datasets/detect/visdrone/) (`VisDrone.yaml`) when their classes match your task. For custom labels (for example traffic signs or interior objects), use the standard YOLO detection layout and point `data=` at your own [dataset YAML](https://docs.ultralytics.com/datasets/). Driving footage often includes small, distant objects; after a baseline run, try a larger `imgsz` if metrics plateau.

Reviewer: 💡 **MEDIUM**: This wording implies users can directly train their own automotive dataset with `kitti.yaml` or `VisDrone.yaml` whenever the classes align. Those dataset YAMLs point to the KITTI/VisDrone data paths, so for a custom dataset the user still needs their own YAML and should treat these files only as templates for class names/structure. Please clarify that distinction to avoid readers accidentally training on the wrong dataset or getting path errors.

**Suggested change:**
```suggestion
    For vehicle-centric detection, you can use configs such as [KITTI](https://docs.ultralytics.com/datasets/detect/kitti/) (`kitti.yaml`) or [VisDrone](https://docs.ultralytics.com/datasets/detect/visdrone/) (`VisDrone.yaml`) as templates when their classes match your task. For custom data, keep the standard YOLO detection layout and point `data=` to your own [dataset YAML](https://docs.ultralytics.com/datasets/). Driving footage often includes small, distant objects; after a baseline run, try a larger `imgsz` if metrics plateau.
```
